webhook/env_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"os"
+	"testing"
+)
+
+const (
+	testDefaultPort      = 8888
+	testMissingInputName = "MissingInput" // portEnvKey is not found.
```

Comment:
I don't think we need this constant.

Reply:
It's used to determine whether or not to set the portEnvKey env var. The "MissingInput" test case corresponds to current yaml that doesn't contain portEnvKey.

Reply:
I know, but you can just use "Missing" constant inline.
Up to you though.